themes/cities/templates/cities-masthead.handlebars

```diff
@@ -7,5 +7,5 @@
             <li><a href="storyBrowse.html">{{localizedMessages.link_browse_text}}</a></li>
         </ul>
     </nav>
-    <a class="sjrk-st-edit-link" href="{{localizedMessages.editor_url}}">{{localizedMessages.link_edit_text}}</a>
+    {{!--<a class="sjrk-st-edit-link" href="{{localizedMessages.editor_url}}">{{localizedMessages.link_edit_text}}</a>--}}
```

Comment:
Should remove the commented out code, or provide a comment for why it is needed.